app/views/requests/show.html.erb

```diff
@@ -55,22 +55,25 @@
         </div>
       <% end %>
     <% else %>
+      <% if @request.creator == current_user %>
```

Comment:
We need a secondary check here to determine if the current user is the helper for that request since they shouldn't see the start conversation button if they've already been assigned as the helper. Something like:

`<% if @request.creator == current_user || current_user == @request.helper %>`